underfs/pom.xml

```diff
@@ -34,6 +34,7 @@
     <module>wasb</module>
     <module>web</module>
     <module>ozone</module>
+    <module>b2</module>
```

Comment:
Could you move this alphabetically in the list? (also the existing `ozone` entry)

Reply:
done